zebra-consensus/src/block.rs

```diff
@@ -53,8 +53,15 @@ pub(crate) fn node_time_check(
     }
 }
 
+/// Check that there is exactly one coinbase transaction in `Block`, and that
+/// the coinbase transaction is the first transaction in the block.
```

Comment:
```suggestion
/// Returns `Ok(())` if there the first transaction in the block is the only
/// coinbase transaction in the block.
```

Reply:
Question: Is "only" ambiguous here?

For example, does it include a block with no coinbase transactions?

```
/// Returns `Ok(())` if this block has one coinbase transaction, and it is the first transaction in the block.
```

Reply:
whats the alternate interpretation?

Reply:
I think the original wording was clearer.

Reply:
> whats the alternate interpretation?

It's unclear what the function returns when the block has no coinbase transactions.